src/vs/platform/webview/electron-main/webviewMainService.ts

```diff
@@ -57,4 +62,43 @@ export class WebviewMainService extends Disposable implements IWebviewManagerSer
 			contents.setIgnoreMenuShortcuts(enabled);
 		}
 	}
+
+	public async findInFrame(windowId: WebviewWindowId, frameName: string, text: string, options: { findNext?: boolean, forward?: boolean }): Promise<void> {
+		const frame = this.getFrameByName(windowId, frameName);
+		frame.findInFrame(text, {
```

Comment:
I think these methods do not exist in the public `electron.d.ts` so you will have to do function probing and casting similar to what I have done here:

https://github.com/microsoft/vscode/blob/2cb7b42d98e44fade2067f06f61c3cf5d1defd58/src/vs/code/electron-main/app.ts#L150-L165